pkg/vfs/assets.gen.go

```diff
@@ -32733,6 +33741,74 @@ func profilesSdsYaml() (*asset, error) {
 	return a, nil
 }
 
+var _profilesSdsYamlOrig = []byte(`apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+spec:
+  security:
+    components:
+      nodeAgent:
+        enabled: true
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+=======
+>>>>>>> 0584d5d7... add back the mandiff test. (#223)
```

Comment:
this doesn't seem right?

Reply:
no, it doesn't seem right - love the understatement :-)
this was just additional cruft that appeared because of some .orig files being present in the source dir, these are gitignored. thanks.